lib/manager/flux/readme.md

```diff
@@ -1,6 +1,9 @@
-Checks YAML manifests for [Flux](https://fluxcd.io/) `HelmRelease` resources and extracts dependencies for the `helm` datasource.
+This manager parses [Flux](https://fluxcd.io/) YAML manifests and:
```

Comment:
We should note, that this currently isn't supported on renovate full docker image

 - renovatebot/docker-renovate-full#406

Reply:
👍 I think the note I just added to the docs should cover this.